### PR TITLE
Changes in theme customizer as per wordpress.org guidelines

### PR DESCRIPTION
**Pull request by ujjawalag**

### CODE REVIEW

File: includes/customizer.php [hunk 1]
```diff
@@ -0,0 +1,340 @@
...
+
+	$wp_customize->add_section( 'responsive_social_media', array(
+		'title'             => __( 'Social Icons', 'responsive' ),
+		'priority'          => 40,
+		'description'       => __( 'Enter the URL to your account for each service for the icon to appear in the header.', 'mobilefirst' ),
+	) );
+
+	// Add Twitter Setting
+
+	$wp_customize->add_setting( 'responsive_theme_options[twitter_uid]', array( 'sanitize_callback' => 'esc_url_raw', 'type' => 'option' ) );
```
Comment (grappler) [inline includes/customizer.php]: It seems that `sanitize_callback` is not a valid argument here.

**Comment by grappler** [comment]
It looks like only the Social Media section has been sanitized and not the other sections.

Is there are reason why you did not update the previous pull request? It can be easier to track the changes that way.

**Comment by ujjawalag** [comment]
I have added 'responsive' prefix to all the sanitation function names.

**Comment by ujjawalag** [comment]
sanitize_text_field() function is used for text fields and I have created custom sanitize textarea function for inline JS and CSS

**Comment by grappler** [comment]
Ok, the functions that you should use for the CSS and JS are either `wp_filter_nohtml_kses()` or `wp_strip_all_tags()`

**Comment by ujjawalag** [comment]
Changed sanitation function for inline CSS and JS

**Pull request merged**